.github/workflows/php.yaml

```diff
@@ -36,3 +36,57 @@ jobs:
       - uses: gaurav-nelson/github-action-markdown-link-check@v1
         with:
           use-verbose-mode: 'yes'
+
+  unit-tests:
+    name: "Unit tests"
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['5.6', '7.1', '7.2', '7.3', '7.4']
+        dependencies: ['']
+        include:
+          - { php-version: '5.6', dependencies: '--prefer-lowest' }
+          - { php-version: '8.0', dependencies: '--ignore-platform-req=php' }
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: mbstring, intl, zip
+          coverage: xdebug
+          ini-values: ${{ matrix.xdebug-ini-values }}
+          tools: composer:v2
+
+      - name: Install dependencies
+        run: composer update --no-interaction ${{ matrix.dependencies }}
+
+      - name: Apply PHPUnit pathes
+        if: ${{ matrix.php-version < 7.2 }}
+        run: scripts/apply-phpunit-patches.sh
+
+      - name: Run tests
+        run: vendor/bin/phpunit --testsuite unit --colors=always --coverage-clover ./logs/clover.xml
+
+      - name: Submit coverage to Coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COVERALLS_PARALLEL: true
+          COVERALLS_FLAG_NAME: ${{ github.job }}-PHP-${{ matrix.php-version }} ${{ matrix.dependencies }}
+        run: |
+          composer global require php-coveralls/php-coveralls
```

Comment:
why not using the action for that ?

Reply:
I'm not sure the official coveralls action supports clover reports: https://github.com/coverallsapp/github-action/issues/15

Do you use it anywhere?

Reply:
Good point, maybe phpunit would have another report type that would be compatible
Maybe open an issue so they can know we need support for clover files

Reply:
And we use the action to mark as finished the uploads
what gives some confusion about why we do not use it here also 
An issue number as a comment would be perfect :)

Reply:
Well the https://github.com/coverallsapp/github-action/issues/15 is exactly about this (clover support). I can add the reference there.

The action for "mark as finished" is used because I wasn't able to make it working via curl, [as mentioned here](https://docs.coveralls.io/parallel-build-webhook), and the action just works.